ocp-templates/app-interface/scheduler.yaml

```diff
@@ -0,0 +1,136 @@
+        - env:
+          - name: POSTGRESQL_HOST
+            valueFrom:
+              secretKeyRef:
+                key: db.host
+                name: ibutsu-db
+          - name: POSTGRESQL_PORT
+            valueFrom:
+              secretKeyRef:
+                key: db.port
+                name: ibutsu-db
+          - name: POSTGRESQL_USER
+            valueFrom:
+              secretKeyRef:
+                key: db.user
+                name: ibutsu-db
+          - name: POSTGRESQL_PASSWORD
+            valueFrom:
+              secretKeyRef:
+                key: db.password
+                name: ibutsu-db
+          - name: POSTGRESQL_DATABASE
+            valueFrom:
+              secretKeyRef:
+                key: db.name
+                name: ibutsu-db
+          - name: REDIS_PASSWORD
+            valueFrom:
+              secretKeyRef:
+                key: db.password
+                name: redis
+                optional: true
+          - name: REDIS_HOSTNAME
+            valueFrom:
+              secretKeyRef:
+                key: db.endpoint
+                name: redis
+          - name: REDIS_PORT
+            valueFrom:
+              secretKeyRef:
+                key: db.port
+                name: redis
+          - name: CELERY_BROKER_URL
+            value: redis://:${REDIS_PASSWORD}@${REDIS_HOST}
+          - name: CELERY_RESULT_BACKEND
+            value: redis://:${REDIS_PASSWORD}@${REDIS_HOST}
+          - name: FRONTEND_URL
+            value: https://${FRONTEND_ROUTE}
+          - name: BACKEND_URL
+            value: https://${BACKEND_ROUTE}
+          image: quay.io/ibutsu/scheduler:${IMAGE_TAG}
+          imagePullPolicy: IfNotPresent
+          name: ibutsu-scheduler
+          resources: {}
+          terminationMessagePath: /dev/termination-log
+          terminationMessagePolicy: File
+        dnsPolicy: ClusterFirst
+        restartPolicy: Always
+    triggers:
+    - type: ConfigChange
+# -----------------------------------------------
+- kind: Service
+  apiVersion: v1
+  metadata:
+    labels:
+      app: ${APP_NAME}
+    name: ibutsu-scheduler
+    namespace: ${NAMESPACE}
+  spec:
+    ports:
+    - port: 8080
+      targetPort: 8080
+    selector:
+      deploymentConfig: ibutsu-scheduler
+# ===============================================
+# Parameters
+# ===============================================
+parameters:
+- name: APP_NAME
+  displayName: App Name
+  description: The name of the application
+  value: ibutsu-server
+- name: NAMESPACE
+  displayName: Namespace
+  description: The namespace for all of the images, applications, etc.
+  value: iqe-ibutsu-server
+- name: BACKEND_ROUTE
+  displayName: Backend Route
+  description: The URL of the backend of the Ibutsu server
+  value: ibutsu-api.example.com
+- name: FRONTEND_ROUTE
+  displayName: Frontend Route
+  description: The URL of the frontend of the Ibutsu server
+  value: ibutsu.example.com
+- name: REDIS_PASSWORD
```

Comment:
Same here -- I think you're no longer using this param since you're fetching the value from a secret.